src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr

```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
-    file source=$(OutputPath)\csc.exe vs.file.ngen=yes
-    file source=$(OutputPath)\csi.exe vs.file.ngen=yes
-    file source=$(OutputPath)\vbc.exe vs.file.ngen=yes
+    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngenArchitecture=all vs.file.ngenPriority=1
```

Comment:
Should the priority be 3 instead? 1 means the NGEN operations will be done immediately, which will increase installation time. vs.file.ngen=yes has the default value of 3.